selfdrive/test/test_onroad.py

```diff
@@ -112,7 +112,7 @@ def check_cpu_usage(first_proc, last_proc):
       cpu_usage = cpu_time / dt * 100.
       if cpu_usage > max(normal_cpu_usage * 1.15, normal_cpu_usage + 5.0):
         # cpu usage is high while playing sounds
-        if proc_name == "./_soundd" and cpu_usage < 25.:
+        if proc_name == "./_soundd" and cpu_usage < 65.:
```

Comment:
want to look into this weird CPU usage? maybe it has to do with the audio backend since this doesn't seem to happen on c3? c2 uses the opensl es backend